internal/transformer/k8stransformer.go

```diff
@@ -218,9 +214,9 @@ func (kt *K8sTransformer) WriteObjects(outputPath string, transformPaths []strin
 	return nil
 }
 
-func (kt *K8sTransformer) generateHelmMetadata(outputPath string, values outputtypes.HelmValues) error {
+func (kt *K8sTransformer) generateHelmArtifacts(outputPath string, values outputtypes.HelmValues, transformPaths []string) error {
 	deployPath := filepath.Join(outputPath, common.DeployDir)
-	helmPath := filepath.Join(deployPath, "helm")
+	helmPath := filepath.Join(deployPath, "helm", kt.Name)
```

Comment:
Lets call it "helm-charts". Probably we can add a helm chart for the ci/cd artifacts too.

Reply:
We already have CI/CD don't we?

Reply:
`Lets call it "helm-charts"` done